exporter/pwd_file.go

```diff
@@ -0,0 +1,28 @@
+package exporter
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type PasswordMap struct {
+	RedisPwd map[string]string
+}
+
+func NewPasswordMap() *PasswordMap {
+	return &PasswordMap{}
+}
+
+func (p *PasswordMap) LoadPwdFile(pwdFile string) {
```

Comment:
Thinking about this more, it's probably a bit more clear flow if this returns as error and the caller logs the error and calls log.Fatalf()